config.suggest

```diff
@@ -128,25 +128,24 @@ EOF
 
 chaos5_mpiicc14_cxx11_mpi2) # CHAOS5, Intel v. 14 compilers with LLNL mpiicpc front ends, using C++ 11 standard and MPI2
 cat <<-EOF
-	CXXFLAGS="-mpi=mvapich2-intel-1.9 -std=c++11"
+	CXXFLAGS="-mpi=mvapich2-intel-1.9 -std=c++11 -gnu-prefix=/usr/apps/gnu/4.9.3/bin/ -Wl,-rpath,/usr/apps/gnu/4.9.3/lib64"
 	CFLAGS=-mpi=mvapich2-intel-1.9
-        FFLAGS=-mpi=mvapich2-intel-1.9
-        --with-mpi-include=/usr/local/tools/mvapich2-intel-1.9/include
-        --with-mpi-lib-dirs=/usr/local/tools/mvapich2-intel-1.9/lib
-        --with-mpi-libs="mpich fmpich"
+	FFLAGS=-mpi=mvapich2-intel-1.9
+	--with-mpi-include=/usr/local/tools/mvapich2-intel-1.9/include
+	--with-mpi-lib-dirs=/usr/local/tools/mvapich2-intel-1.9/lib
+	--with-mpi-libs="mpich fmpich"
 	--with-CC=mpiicc
 	--with-CXX=mpiicpc
 	--with-F77=mpiifort
 	--without-petsc
 	--without-x
 	--without-sundials
 	--disable-shared
-	--enable-rpath=-Wl,-R
 	--with-zlib
-	--with-hdf5=/collab/usr/gapps/samrai/apps/chaos/hdf5/1.8.5p1-serial
-	--with-silo=/usr/gapps/silo/4.8/chaos_5_x86_64_ib
-	--with-hypre=/collab/usr/gapps/samrai/devapps/chaos/hypre/2.10.0b.opt-mvapich2
-	--with-boost=/collab/usr/gapps/samrai/devapps/chaos/boost/1.58.0
+	--with-hdf5=/collab/usr/gapps/samrai/devapps/linux-rhel6-x86_64/gcc-4.9.3/hdf5-1.8.12-b3ywl76b5yl7xpq45br24im5filadcrq
```

Comment:
Are these the libraries that I built?

Reply:
Yes they are, but I was able to compile/run with the original hdf5 installation.  Let me check config.suggest again before I merge this.

Reply:
I am going to revert these flags, as I have verified that the installations we were already using still work.

Reply:
Oh that's cool. Do they work at runtime as well as compile time?

Reply:
Yes, I ran the full test suite.